api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3590,6 +3590,12 @@ message LockDelete {
     (gogoproto.embed) = true,
     (gogoproto.jsontag) = ""
   ];
+
+  // Target describes the set of interactions that the lock applies to
+  types.LockTarget Target = 4 [
```

Comment:
Should we factor this out into a separate `LockMetadata` that we can then reuse in different lock-related audit events?

Reply:
I can, but I'm not sure what else needs to be added to `LockMetadata` that isn't already present in `LockTarget`, `UserMetadata` or `ResourceMetadata` unless you'd like those 3 messages rolled into `LockMetadata`